queue_processors/queue_processor/reduction/runner.py

```diff
@@ -0,0 +1,152 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+# !/usr/bin/env python
+# pylint: disable=broad-except
+# pylint: disable=bare-except
+"""
+Post Process Administrator. It kicks off cataloging and reduction jobs.
```

Comment:
Should all references to `Post Process Administrator` and `Post Process admin` be removed. The name of this file is `runner.py` perhaps a good name, but the connection it name to comment `It kicks off cataloging and reduction jobs` is not clear to me